plugins/wfscroll.json

```diff
@@ -0,0 +1,8 @@
+{
+    "id":"153",
+    "title":"wfscroll: Animate on scroll",
```

Comment:
Since your 2 others plugins have the "wfs" in uppercase, for consistency WFScroll would be better (you even wrote it like that in the description)